### PR TITLE
Added Apna-College-Clone

### PR DESCRIPTION
**Pull request by heli0707**
Closes: #466 

- **Title** : Apna-College-Clone
- **Name:** : Heli
- **Idenitfy yourself:** : GSSOC Contributor

### Describe the add-ons or changes you've made 📃
The project added as Apna-College-Clone
The clone project uses HTML, CSS, and JavaScript to create reusable components for better maintainability and scalability. It manages routes for "Home", "New Courses", "Log In", and "Sign Up" pages, and includes forms with basic validation. The design ensures consistency and responsiveness across the webpage.

### How Has This Been Tested? ⚙️
I have tested it on my own.

### Screenshots 📷
s1
s2
s3

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by heli0707** [comment]
@Avdhesh-Varshney 
Please review it and let me know if there are any problems with it

File: .vscode/settings.json
Comment (Avdhesh-Varshney) [inline .vscode/settings.json:1]: revoke these changes

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@heli0707 enlist your project in the readme file of vanilla js

**Review by heli0707 — COMMENTED** [review]
settings.json file updated 
review it!

**Comment by heli0707** [comment]
@Avdhesh-Varshney 
listed project in the readme file of vanilla js
could you see that?

File: Vanilla-JS-Projects/Basic/Apna-College-Clone/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<h1 align='center'><b>💥 Isogram Checker 💥</b></h1>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Apna-College-Clone/README.md:1]: carefully write the readme.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Basic/Apna-College-Clone/README.md (hunk 1 above)
Comment (heli0707) [reply Vanilla-JS-Projects/Basic/Apna-College-Clone/README.md]: done

**Review by heli0707 — COMMENTED** [review]
@Avdhesh-Varshney 
done all changes

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**